store/rootmulti/store_test.go

```diff
@@ -617,6 +623,10 @@ func TestMultiStore_PruningRestart(t *testing.T) {
 	actualHeightToPrune = ms.pruningManager.GetPruningHeight(ms.LatestVersion())
 	require.Equal(t, int64(8), actualHeightToPrune)
 
+	// Ensure async pruning is done
+	time.Sleep(500 * time.Millisecond)
+	ms.Commit()
+
```

Comment:
Tests are passing without `PausePruning` set. How can you ensure the expected behaviour?